src/target/riscv/riscv-013.c

```diff
@@ -2001,6 +2001,226 @@ static int riscv013_set_register_buf(struct target *target,
 	return result;
 }
 
+/**
+ * @par size in bytes
+ */
+static void write_to_buf(uint8_t *buffer, uint64_t value, unsigned size)
```

Comment:
Can already existing `buf_set_u64()` function be used instead?

Reply:
Done. Thanks.